AK/ByteBuffer.h

```diff
@@ -192,11 +192,14 @@ class ByteBuffer {
         MUST(append(other.data(), other.size()));
     }
 
-    void overwrite(size_t offset, void const* data, size_t data_size)
+    ErrorOr<void> overwrite(size_t offset, void const* data, size_t data_size)
     {
         // make sure we're not told to write past the end
-        VERIFY(offset + data_size <= size());
+        if (offset + data_size > size())
+            return Error::from_string_literal("ByteBuffer::overwrite: out of bounds");
```

Comment:
No.
This is not a simple runtime error, this is a hard logical error.
Please keep this as a VERIFY.

Reply:
You're right, this is pretty stupid - I honestly don't really know what I was thinking. I'll take a look at your suggested changes later today, but I'm thinking this PR might just need to be axed.

Reply:
I think I will just close this pull request. I'm sorry to have wasted your time